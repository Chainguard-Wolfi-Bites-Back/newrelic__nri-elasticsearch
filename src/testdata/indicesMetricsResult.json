[
	{
		"health": "green",
		"status": "open",
		"index": "twitter",
		"uuid": "rj_6yR2fSLWNxMHvw7QSiw",
		"pri": "3",
		"rep": "2",
		"docs.count": "1",
		"docs.deleted": "0",
		"store.size": "12kb",
		"pri.store.size": "4kb"
	},
	{
		"health": "yellow",
		"status": "open",
		"index": "reddit",
		"uuid": "im-Uv5zrTYKs7Mtj6-478A",
		"pri": "7",
		"rep": "3",
		"docs.count": "1",
		"docs.deleted": "0",
		"store.size": "15.3kb",
		"pri.store.size": "5.1kb"
	}
]