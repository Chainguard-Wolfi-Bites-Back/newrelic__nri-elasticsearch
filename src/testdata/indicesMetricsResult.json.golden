{"displayName":"rj_6yR2fSLWNxMHvw7QSiw","entityName":"index:rj_6yR2fSLWNxMHvw7QSiw","event_type":"ElasticsearchIndexSample","index.docs":"1","index.docsDeleted":"0","index.health":"green","index.primaryShareds":"3","index.primaryStoreSizeInBytes":"4kb","index.replicaShards":"2","index.storeSizeInBytes":"12kb"}